### PR TITLE
Sp 1

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

File: src/main/java/spring/intro/model/User.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    @Override
+    public String toString() {
+        return "User{"
+                + "id=" + id + ", email='" + email + '\''
+                + ", password='" + password + '\'' + '}';
```
Comment (AndrewPolishchuc) [inline src/main/java/spring/intro/model/User.java:48]: It will be better to hide this field

**Review by AndrewPolishchuc — COMMENTED** [review]
Good job!

File: src/main/java/spring/intro/model/User.java (hunk 1 above)
Comment (andromash) [reply src/main/java/spring/intro/model/User.java:48]: Well, it is not a secured project. It is just an example to see how everything works

File: src/main/java/spring/intro/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<User> getAll() {
+        try (Session session = sessionFactory.openSession()) {
+            Query<User> getAllUsers = session.createQuery("from User", User.class);
+            return getAllUsers.getResultList();
+        } catch (Exception e) {
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/dao/UserDaoImpl.java]: catching exception is optional with read operations

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**